fullstack/task/packages/server/src/entities/exchange-rate.entity.ts

```diff
@@ -0,0 +1,42 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+import { IsString, IsNumber, MinLength } from 'class-validator'; // Import IsNumber decorator
+import { BeforeInsert, BeforeUpdate, Column, Entity } from 'typeorm';
+
+import { EntityWithMeta } from '../common';
+import { VAR_CHAR } from './constants';
+
+@ObjectType()
+@Entity()
+export class ExchangeRate extends EntityWithMeta {
+    @IsString()
+    @MinLength(1)
+    @Field(() => String)
+    @Column({ ...VAR_CHAR, unique: true })
+    public code!: string;
+
+    @IsString()
+    @MinLength(1)
+    @Field(() => String)
+    @Column({ ...VAR_CHAR })
+    public currency!: string;
+
+    @IsString()
+    @Field(() => String)
+    @Column({ ...VAR_CHAR })
+    public country!: string;
+
+    @Field(() => String)
```

Comment:
Here I planned to use a transformer with decimal.js or similar lib, but just wasted a lot of time.